### ISSUE TITLE
The main menu is a bit of a confusing mess

### ISSUE THREAD

**Issue by TwistedHBG** [jasonrohrer/OneLife#1121]
Inspired by this reddit post - https://www.reddit.com/r/onehouronelife/comments/1etft9n/how_do_i_join_a_server/

It's not the first time I've seen people being confused by the minimalistic main menu. I think it was fine when the game first released, but as new features and options got added it's become a bit hard to parse for someone who is seeing it for the first time.

My main issues with it are:

**1. The 'lives left' counter at the top takes up a lot of space, it doesn't explain itself very well, and it's useless in 99% of situations**

The life limit was originally added so you couldn't just keep dying until you get the life you wanted, but there have been changes to the game that actually allow that (/die now cycles through various families until you try them all and then spawns you in Donkey Town for a life, living to 60 lets you respawn in a family where you just were).

I am not against having a life limit by itself (even though I don't think it really serves a purpose anymore), but having the counter there is extra visual noise and information to parse, and since it is almost never relevant it can easily cause confusion.

I've seen people that think they can only ever play 48 lives of the game, ever.
I've seen people that though this was actually the server status, and they thought the servers were full at 48/48 people
I've seen people that thought they had to wait 5 minutes before playing because the servers are full
And I've seen people who were just confused at the meaning of that big, prominent block of text at the very block of the screen because it never seemed relevant but it is so front and center.

I think those two lines should be removed, or at least made invisible until the number goes below some threshold.

**2. The 'LOGIN' button, the main button that you need to play the game, is kinda hiding off to the side and it's not clear that's how you PLAY the game**

People associate 'login' buttons with typing in their login information. When I see a login button, I assume clicking it will take me to a page where I can enter or re-enter my username or password.

It made more sense before the game was available on Steam, when you had to actually enter your email and key and then press 'login' before playing for the first time, but when people buy the game on Steam they probably don't even realize that an account is made for them and the login information is entered automatically.

I would personally rename LOGIN to PLAY, because that better reflects what the button actually does

Furthermore, right now the button is kinda off to the side with several other buttons and it doesn't seem important, even though it it THE main button of the OHOL menu. Moving the GENES/FRIENDS/FAMILY TREES/TECH buttons a bit lower so the LOGIN/PLAY button can stand alone would both make it clearer and look nicer.

Also, I think moving the account info to its separate menu would look make the whole thing nicer as it's something the average player never interacts with and it takes up a lot of room.

I'm adding in some images for comparison with the current menu and maybe inspiration

![OHOLMockup](https://github.com/user-attachments/assets/617e57be-7cc9-4016-bd92-548762054f93)
![OHOLMockup2](https://github.com/user-attachments/assets/1245104d-b0d6-473e-8991-1b77dfa4a865)

**Comment by jasonrohrer** [comment]
Great stuff!

That Login screen was definitely "grown" and not "designed".

It does seem that the "Enter Login Info" screen should maybe pop up once if needed (if launching the game off-Steam for the first time) and then not be shown after that.

**Comment by jasonrohrer** [comment]
I made some progress here.

I don't want to completely overhaul this screen, and it is important that people are aware that they are actually logging into a server, and that their account details are important.  It is an online-only game, and that should be crystal clear.

The lives left thing is mostly vestigial, so it's hidden now unless the user is low on lives.  The login button is highlighted now, which should draw attention to it for anyone who is confused.
